action.yml

```diff
@@ -97,4 +111,4 @@ runs:
         github-token: ${{ inputs.github-token }}
         script: |
           const script = require('${{ github.action_path }}/dist/index.js')
-          await script({ github, context, inputs: ${{ toJSON(inputs) }} })
+          await script({ github, context, inputs: ${{ toJSON(inputs) }}, packageVersion: "${{ steps.get-package-version.outputs.PACKAGE_VERSION }}", packageName: "${{ steps.get-package-name.outputs.PACKAGE_NAME }}" })
```

Comment:
ultimately, and to summarize my earlier comments, isn't it just simpler to grab package version and name from within the code than here in the yaml?